src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+### Cryptography & ZKP
+
+Zero-knowledge proofs (ZKP) and cryptography are critical for building privacy and security into Ethereum and its applications. Zero-knowledge is a relatively young but fast-moving space with many open research and development opportunities. Some possibilities include developing more efficient implementations of the [Keccak hashing algorithm](https://hackmd.io/sK7v0lr8Txi1bgION1rRpw?view#Overview), finding better polynomial commitments than currently exist or reducing the cost of ecdsa public key generation and signature verification circuits.
+
+#### Background reading
+
+[0xparc blog](https://0xparc.org/blog)
+
+[zkp.science](https://zkp.science/)
+
+[Zero Knowledge podcast](https://zeroknowledge.fm/)
+
+
+#### Recent research
+
+[Recent advance in elliptic curve cryptography](https://ethresear.ch/t/the-ec-fft-algorithm-without-elliptic-curve-and-isogenies/11346)
+
+[Ethresear.ch ZK](https://ethresear.ch/c/zk-s-nt-arks/13)
+
+
+### Wallets
+
+User-wallets can be browser extensions, desktop and mobile apps or smart contracts on Ethereum. There is active research into social recovery wallets that reduce some of the risk associated with individual-user key management. Associated with dveelopment of wallets is research into alternative forms of account abstraction, which is an important area of nascent research.
+
+#### Background reading
+
+[Our Security page](https://ethereum.org/en/security/)
+
+[ethresear.ch Security](https://ethresear.ch/tag/security)
+
+[EIP-2938 Account Abstraction](https://eips.ethereum.org/EIPS/eip-2938)
+
+[EIP-4337 Account Abstraction](https://eips.ethereum.org/EIPS/eip-4337)
+
+
+#### Recent research
+
+[Validation focussed smart contract wallets](https://ethereum-magicians.org/t/validation-focused-smart-contract-wallets/6603)
+
+[The future of accounts](https://ethereum-magicians.org/t/validation-focused-smart-contract-wallets/6603)
+
+[EIP-3074](https://eips.ethereum.org/EIPS/eip-3074)
+
+[Publishing code code at EOA addresses](https://github.com/ethereum/EIPs/pull/5003)
+
+
+## Community, education and outreach
+
+Onboarding new users onto Ethereum requires new educational resources and approaches to outreach. This might include blog posts and articles, books, podcasts, memes, teaching resources events and anything else that builds communities, welcomes new starters and educates people about Ethereum.
+
+### UX/UI
+
+For Ethereum to onboard more people the UX/UI must be improved. This requires designers and UI and product experts to re-examine the design of wallets and apps.
+
+#### Background reading
+
+[Ethresear.ch UX/UI](https://ethresear.ch/c/ui-ux/24)
+
+[Ethereum.org community page](https://ethereum.org/en/learn/)
```

Comment:
Fix link: https://ethereum.org/en/contributing/style-guide/#internal-links

Reply:
I might also remove this link - I don't see any mention of UI/UX on this page https://ethereum.org/en/learn/

Reply:
I might also remove this link - I don't see any mention of UI/UX on this page https://ethereum.org/en/learn/